posthog/queries/test/test_lifecycle.py

```diff
@@ -25,7 +25,17 @@ def create_action(**kwargs):
     return action
 
 
-class TestLifecycle(ClickhouseTestMixin, APIBaseTest):
+class TestLifecycleBase(ClickhouseTestMixin, APIBaseTest):
+    def assertLifecycleResults(self, results, expected):
+        sorted_results = [
+            {"status": r["status"], "data": r["data"]} for r in sorted(results, key=lambda r: r["status"])
+        ]
+        sorted_expected = list(sorted(expected, key=lambda r: r["status"]))
+
+        self.assertListEqual(sorted_results, sorted_expected)
+
+
```

Comment:
I'm not sure why the lifecycle query had to change in this retention query PR? Something to do with the query date range?